build/images/codegen/Dockerfile

```diff
@@ -50,7 +50,7 @@ RUN go install k8s.io/code-generator/cmd/client-gen@kubernetes-$K8S_VERSION && \
     go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.33.0 && \
     go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.3.0 && \
     go install golang.org/x/tools/cmd/goimports@latest && \
-    go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.0
+    go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.14.0
```

Comment:
We need a new version of codegen due to this update, @antoninbas I suppose you need to push it locally? I am thinking maybe we can have a workflow to handle this image update in the future?
@shikharish please help to update this document as well to reflect the new version: https://github.com/antrea-io/antrea/tree/main/build/images/codegen

```
| kubernetes-1.29.2-build.1      | Upgraded controller-gen to v0.14.0 |
```

Reply:
I pushed `antrea/codegen:kubernetes-1.29.2-build.1` to the registry

@shikharish you will need to change all `antrea/codegen:kubernetes-1.29.2-build.0` references to use the new tag (`antrea/codegen:kubernetes-1.29.2-build.1`), and also update the README as per Lan's suggestion

Reply:
After updating the tag everywhere, you will also need to run `make codegen` and commit the changes.
You may need to run `make codegen` at the root of the repo AND in the `multicluster/` folder.